### DISCUSSION TITLE
How to render nodes after updating storage?

### DISCUSSION THREAD

**Discussion by yinran100** [ueberdosis/tiptap#2314] [General Questions]
How to contact `Storage` and `Node` ？
I want to be able to update all the `Node` in tiptap with `commands`

**Comment by philippkuehn** [comment]
I don’t know exactly what you are trying to do. Please explain in more detail what you want and what you tried.

**Reply by yinran100** [reply]
I have a `CommentPlugins`, it is a `Node` , Now I know how to add comment，all comments share data in  `Storage`，Each comment has its own `attr`, which fetches data from the `Storage`，and I need a method to remove the specified comment.
My idea was to update `Storage` and then trigger the update of Node, but it didn't work.  （If I don't edit this comment, it won't render again.）
![image](https://user-images.githubusercontent.com/38638816/147397102-2196f697-ef6c-449b-8851-a27f834a23e4.png)

I'm not sure there's a better way. I don't know to update the node without selecting it.
